cpp/log/database_test.cc

```diff
@@ -47,7 +47,12 @@ typedef testing::Types<FileDB<cert_trans::LoggedCertificate>,
 
 typedef Database<cert_trans::LoggedCertificate> DB;
 
+template <class T>
+class DBTestDeathTest : public DBTest<T> {
```

Comment:
Could use the newfangled way here, if you wanted (but this is reasonable too, I think):

```C++
template <typename T>
using DBDeathTest = DBTest<T>;
```